examples/aliasfree/demo_alias_free.py

```diff
@@ -0,0 +1,138 @@
+import deepinv as dinv
+from deepinv.loss import adversarial
+from deepinv.physics.generator import MotionBlurGenerator
+import torch
+from torch.utils.data import DataLoader, random_split
+from torchvision.datasets import ImageFolder
+from torchvision.transforms import Compose, ToTensor, CenterCrop, Resize
+from torchvision.datasets.utils import download_and_extract_archive
+from torchvision.transforms import InterpolationMode
+
+import numpy
+import random
+
+from os.path import exists
+
+torch.manual_seed(0)
```

Comment:
I don't have a strong opinion about that but here are some thoughts.

There should be a way to make everything deterministic in deepinverse, ideally in a straightforward, standard and properly documented manner.

I'm not sure if passing around instances of `torch.Generator` is the perfect solution: how do you deal with the rng from numpy? from the stdlib?

I like the idea of acknowledging some rngs as officially supported by deepinverse and relying on their global state but I reckon it might be problematic for parallel computing.

I'd favor forking rng states over re-seeding rngs as it minimizes interference with end-users' code (it makes the entropy higher too).

I like the idea of having an internal module for messing with the rng but I'm not sure I would expose it to end-users, it might not be the best for interoperability with other libraries.

Hope that helps!